ansible_collections/arista/avd/roles/eos_config_deploy_cvp/docs/avd-to-cvaas.md

```diff
@@ -0,0 +1,169 @@
+  and `ansible_password` will reference the service account token, e.g.: `"{{ lookup('file', '/tokens/cvaas.tok')}}"`
+    - reference: [ansible-cvp authentication](https://cvp.avd.sh/en/stable/docs/how-to/cvp-authentication/)
+
+## Steps to create service accounts on CloudVision
+
+1. Go to Settings and Tools --> Access Control --> Service Accounts --> click `+ New Service Account`
+
+```text
+Account name: AVD
+Description: "Automation with AVD"
+Give a description under "Generated Service Account Token"
+Specify the "valid until" date.
+Make sure to copy the generated password. You only get view it once.
+Click "Save" to exit the dialogue box.
+```
+
+![Figure: 1](../../../media/serviceaccount1.png)
+![Figure: 2](../../../media/serviceaccount2.png)
+![Figure: 3](../../../media/serviceaccount3.png)
+
+!!! note
+    The name of the service account must match a username configured to be authorized on
+    EOS, otherwise device interactive API calls might fail due to authorization denial.
+
+## Add CVAAS to your Ansible inventory file
+
+Below is an example, the CVaaS group should be added outside of any group targeted by the AVD `eos_designs` and `eos_cli_config_gen` roles.
+
+```text
+DC1:
+  children:
+    CVAAS:
+      hosts:
+        cvaas:
+```
+
+## Adding group variables for the CVaaS instance
+
+Create a folder under `group_vars` named `CVAAS` and add a file named `cvaas_auth.yml`. Your file should look similar to this:
+
+```text
+ansible_host: www.arista.io
+ansible_user: cvaas
+# Good until 1/24/2030 <update to expiration date of token that was generated in CVaaS>
+# It is advised to use Ansible Vault for storing the `ansible_password`.
+
+ansible_password: <service account token> (generated in CVP when AVD service account was created)
+ansible_connection: httpapi
+ansible_network_os: eos
+ansible_httpapi_use_ssl: True
+ansible_httpapi_validate_certs: True
+ansible_httpapi_port: 443
+```
+
+Click [here](#ansible-vault) for instructions on how to setup Ansible Vault.
+
+## Testing connectivity and authentication between AVD and CVaaS
+
+```text
+root@6e3d94f50dca:/workspace# ansible-playbook playbooks/cvaas_facts.yml
+
+PLAY [Playbook to demonstrate cv_container module.] *********************************************************************************************************************************
+
+TASK [Gather CVaaS facts from cvaas] ************************************************************************************************************************************************
+ok: [cvaas]
+
+PLAY RECAP **************************************************************************************************************************************************************************
+cvaas                      : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+## Sample playbook cvaas_facts.yml
+
+```text
```

Comment:
```suggestion
```yaml
```

Reply:
I removed this section since we are running the ad-hoc command.